kclvm/tools/src/langserver/WordMap.rs

```diff
@@ -0,0 +1,128 @@
+// maybe unused
+use crate::langserver;
+use crate::util;
+use kclvm_error::Position;
+use std::collections::HashMap;
+
+// this class maybe replaced by scope in sema
+#[derive(Default)]
+pub struct FileWordMap {
```

Comment:
waiting for more doc and comment on APIs

Reply:
comments can be seen in front of functions